python_modules/dagster/dagster/core/storage/runs/base.py

```diff
@@ -387,3 +387,18 @@ def update_backfill(self, partition_backfill: PartitionBackfill):
 
     def alembic_version(self):
         return None
+
+    # Key Value Storage
+    #
+    # Stores arbitrary key-value pairs. Currently used for the cursor for the auto-reexecution
+    # deamon. Bundled into run storage for convenience.
+
+    @abstractmethod
```

Comment:
seems like this shouldn't be `@abstractmethod` and the other methods (get/set without the actual implementations) should be abstractmethods where the implementing instances raise NotImplementedError